Source/ChocolateyGui/Views/SettingsView.xaml

```diff
@@ -75,14 +75,15 @@
                             <RowDefinition Height="Auto"/>
                             <RowDefinition Height="*"/>
                         </Grid.RowDefinitions>
-                        <TextBlock Margin="5 5 5 3" Style="{StaticResource SubheaderTextBlockStyle}" Text="{x:Static lang:Resources.SettingsView_ChocoGui}"/>
+                        <TextBlock Margin="5 5 5 3" Style="{StaticResource SubheaderTextBlockStyle}" Text="{lang:Localize SettingsView_ChocoGui}" />
                         <ScrollViewer Margin="5 0" Grid.Row="1" Width="300" HorizontalAlignment="Left" VerticalScrollBarVisibility="Auto">
                             <ItemsControl ItemTemplate="{StaticResource ChocolateyGUISettingTemplate}">
                                 <ItemsControl.Items>
-                                    <models:ChocolateyGUISetting Title="{x:Static lang:Resources.SettingsView_ToggleShowConsoleOutputTitle}" Description="{x:Static lang:Resources.SettingsView_ToggleShowConsoleOutputDescription}" Enabled="{Binding Source={StaticResource BindingProxy}, Path=Data.ShowConsoleOutput}" />
-                                    <models:ChocolateyGUISetting Title="{x:Static lang:Resources.SettingsView_ToggleDefaultTileViewLocalTitle}" Description="{x:Static lang:Resources.SettingsView_ToggleDefaultTileViewLocalDescription}" Enabled="{Binding Source={StaticResource BindingProxy}, Path=Data.DefaultToTileViewForLocalSource}" />
-                                    <models:ChocolateyGUISetting Title="{x:Static lang:Resources.SettingsView_ToggleDefaultTileViewRemoteTitle}" Description="{x:Static lang:Resources.SettingsView_ToggleDefaultTileViewRemoteDescription}" Enabled="{Binding Source={StaticResource BindingProxy}, Path=Data.DefaultToTileViewForRemoteSource}" />
-                                    <models:ChocolateyGUISetting Title="{x:Static lang:Resources.SettingsView_UseDelayedSearchTitle}" Description="{x:Static lang:Resources.SettingsView_ToggleUseDelayedSearchDescription}" Enabled="{Binding Source={StaticResource BindingProxy}, Path=Data.UseDelayedSearch}" />
+                                    <metro:SplitButton ItemsSource="{Binding Source={StaticResource BindingProxy}, Path=Data.AllLanguages}" SelectedItem="{Binding Source={StaticResource BindingProxy}, Path=Data.UseLanguage}" />

```

Comment:
If you put  the SplitButton outside the ItemsControl then you don't need the BindingProxy and didn't get this output

```
System.Windows.Data Error: 26 : ItemTemplate and ItemTemplateSelector are ignored for items already of the ItemsControl's container type; Type='SplitButton'
```

Solution

```xaml
<ScrollViewer Margin="5 0" Grid.Row="1" Width="300" HorizontalAlignment="Left" VerticalScrollBarVisibility="Auto">
    <StackPanel Orientation="Vertical">
        <metro:SplitButton ItemsSource="{Binding AllLanguages}" SelectedItem="{Binding UseLanguage, Mode=TwoWay, UpdateSourceTrigger=PropertyChanged}" />
        <ItemsControl ItemTemplate="{StaticResource ChocolateyGUISettingTemplate}">
            <ItemsControl.Items>
                <models:ChocolateyGUISetting Title="{lang:Localize SettingsView_ToggleShowConsoleOutputTitle}" Description="{lang:Localize SettingsView_ToggleShowConsoleOutputDescription}" Enabled="{Binding Source={StaticResource BindingProxy}, Path=Data.ShowConsoleOutput}" />
                <models:ChocolateyGUISetting Title="{lang:Localize SettingsView_ToggleDefaultTileViewLocalTitle}" Description="{lang:Localize SettingsView_ToggleDefaultTileViewLocalDescription}" Enabled="{Binding Source={StaticResource BindingProxy}, Path=Data.DefaultToTileViewForLocalSource}" />
                <models:ChocolateyGUISetting Title="{lang:Localize SettingsView_ToggleDefaultTileViewRemoteTitle}" Description="{lang:Localize SettingsView_ToggleDefaultTileViewRemoteDescription}" Enabled="{Binding Source={StaticResource BindingProxy}, Path=Data.DefaultToTileViewForRemoteSource}" />
                <models:ChocolateyGUISetting Title="{lang:Localize SettingsView_UseDelayedSearchTitle}" Description="{lang:Localize SettingsView_ToggleUseDelayedSearchDescription}" Enabled="{Binding Source={StaticResource BindingProxy}, Path=Data.UseDelayedSearch}" />
            </ItemsControl.Items>
        </ItemsControl>
    </StackPanel>
</ScrollViewer>
```

Reply:
thanks, I didn't notice that error.